### PR TITLE
Feature/fix slow test in mluke

### PR DESCRIPTION
**Pull request by ryokan0123**
# What does this PR do?

Fix slow tests in MLuke/Luke (https://github.com/huggingface/transformers/pull/14690).

* Replace tokenizers to dummy tokenizers in `MLukeTokenizerTest/LukeTokenizerTest`.
* Add a missing entry in toctree.

## Who can review?
@sgugger @NielsRogge

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -661,6 +661,7 @@ Flax), PyTorch, and/or TensorFlow.
     model_doc/mbart
     model_doc/megatron_bert
     model_doc/megatron_gpt2
+    model_doc/mluke
```
Comment (sgugger) [inline docs/source/index.rst:664]: This file is going to be removed and is not used anymore. You should add your model [here](https://github.com/huggingface/transformers/blob/master/docs/source/_toctree.yml)

**Review by sgugger — APPROVED** [review]
This looks like a nice speedup, thanks a lot!
```
slowest durations
0.30s call     tests/test_tokenization_luke.py::LukeTokenizerTest::test_maximum_encoding_length_pair_input
0.19s call     tests/test_tokenization_mluke.py::MLukeTokenizerTest::test_maximum_encoding_length_pair_input
0.19s call     tests/test_tokenization_luke.py::LukeTokenizerTest::test_add_special_tokens
0.18s call     tests/test_tokenization_mluke.py::MLukeTokenizerTest::test_add_special_tokens
0.16s call     tests/test_tokenization_luke.py::LukeTokenizerTest::test_maximum_encoding_length_single_input
0.14s call     tests/test_tokenization_mluke.py::MLukeTokenizerTest::test_maximum_encoding_length_single_input
0.06s call     tests/test_tokenization_mluke.py::MLukeTokenizerTest::test_special_tokens_initialization_with_non_empty_additional_special_tokens
548 durations < 0.05 secs were omitted
```

File: docs/source/index.rst (hunk 1 above)
Comment (Ryou0634) [reply docs/source/index.rst:664]: Oh, I see. Thank you for the pointer!

**Comment by LysandreJik** [comment]
Merging this :)

**Review by LysandreJik — APPROVED** [review]

**Pull request merged**